src/master/develop/connection.md

```diff
@@ -103,14 +105,14 @@ If you're using another development language, please refer to the [immugw](/mast
 
 ::::
 
-<WrappedSection>
-
-### Mutual TLS
+### With Mutual TLS
```

Comment:
This and the `No auth` section below are on different heading level than the `With credentials`